### DISCUSSION TITLE
Filter notifications by org

### DISCUSSION THREAD

**Discussion by dghubble** [community/community#106717] [Mobile]
### Select Topic Area

Product Feedback

### Body

On the GitHub Android app, I'd like to be able to filter notifications by org. Switching between viewing personal and work organization notifications isn't currently possible. On the web there is a filter by org option, it would be nice to see this in the app.

![Screenshot_20240207-205606](https://github.com/community/community/assets/2253428/fa47ec46-584f-498f-8a43-203f84ec5762)

**Comment by HenSquared** [comment]
Nice idea @dghubble , I'll create an issue to discuss and track this internally. 
Thanks for the feedback.
